src/mecab/src/mmap.h

```diff
@@ -107,7 +107,7 @@ template <class T> class Mmap {
 #if 0 /* for Open JTalk */
     hFile = ::CreateFileA(filename, mode1, FILE_SHARE_READ, 0,
```

Comment:
こっちも変えとくのどうでしょう？ 👀 
こう？
```suggestion
    hFile = ::CreateFileA(filename, mode1, FILE_SHARE_READ | FILE_SHARE_DELETE, 0,
```

Reply:
追加しました。